app/assets/javascripts/application.js

```diff
@@ -12,5 +12,25 @@
 //
 //= require rails-ujs
 //= require activestorage
+//= require jquery3
+//= require jquery_ujs
 //= require_tree .
-//= require Chart.bundle.min
```

Comment:
@WHomer Hey let's add this back in, should be good.

Reply:
Yeah, I think that would be a great idea.  Let me consult the QA team to see how this was missed in the first place.

Reply:
@WHomer Addressed in #66.